depthai_cmake/depthai_cmake_macros.cmake.in

```diff
@@ -125,13 +128,10 @@ macro(add_ros_node name)
     add_dependencies(${name}_lib
       ${${name}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 
-    add_library(${name}_nodelet ${_ARG_ROS1_COMPONENT_SOURCES})
-    target_link_libraries(${name}_nodelet ${name}_lib)
+    add_executable(${name}_node ${_ARG_ROS1_EXE_SOURCES})
+    target_link_libraries(${name}_node ${name}_lib)
```

Comment:
```
add_executable(${PACKAGE_NAME}_${name}_node
...
set_target_properties(${PACKAGE_NAME}_${name}_node OUTPUT_NAME ${name}_node)
```

This ensures different packages don't have name collision. Should something like this be considered?